peachjam/admin.py

```diff
@@ -1188,6 +1188,12 @@ def new_init(self, *args, **kwargs):
 class ArticleForm(forms.ModelForm):
     body = forms.CharField(widget=CKEditorWidget())
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        article_tags = Taxonomy.objects.filter(name="Article tags").first()
```

Comment:
There's a helper method on taxonomy that will do this for you

Reply:
Sorry, it's on Article. And it'll create if necessary.

Reply:
added